CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree iOS SDK Release Notes
 
+## Unreleased
```

Comment:
```suggestion
## unreleased
```

^ nit since our [CI is configured to use lowercase](https://github.com/braintree/braintree_ios/blob/6bc35d328d2aa9660d8339e25e14f778925e7811/.github/workflows/release.yml#L22)